.github/workflows/desktop-app-release.yaml

```diff
@@ -0,0 +1,70 @@
+name: Desktop App Release
+
+on:
+  push:
+    branches:
+      - v1.0
+  pull_request:
+    branches:
+      - v1.0 #remove this before committing!
```

Comment:
is this a problem?

Reply:
yeah dont' want it running on every PR? (I guess we could.? means you get a binary for each PR actually... good thinking).